### PR TITLE
docs(enterprise): Update the Crowdsourcing article

### PR DESCRIPTION
**Pull request by rohalskyy**

### CODE REVIEW

File: src/content/docs/enterprise/organization-management/crowdsourcing.mdx [hunk 1]
```diff
@@ -7,62 +7,124 @@ slug: enterprise/crowdsourcing
...
+
+* Name &ndash; Set a custom name for the Crowdsourcing step. The default name is **Crowdsourcing**.
+* Output &ndash; Select the condition of the string to move to the next workflow step. You can choose between **Translated string** and **Translated string with minimum rating of**. For the latter, you can set the required minimum rating for the string before it moves to the next workflow step.
+* Languages &ndash; Select the languages that need translation.
+
+## Crowdsourcing Project Settings
+
+To start configuring your project's Crowdsourcing settings, open your project and go to Workflow > Crowdsourcing. Here, you can manage key aspects, including general project details, Readme content for contributors, status images reflecting localization progress, and the project link to build your community.
+
+Once you’ve configured all Crowdsourcing project settings, click **Publish** to make your project visible on your [organization's public page](#public-organization-settings).
```
Comment (andrii-bodnar) [inline src/content/docs/enterprise/organization-management/crowdsourcing.mdx:56]: Move this sentence after the screenshot

File: src/content/docs/enterprise/organization-management/crowdsourcing.mdx [hunk 2]
```diff
@@ -7,62 +7,124 @@ slug: enterprise/crowdsourcing
...
+
+- **Continuous feedback and improvements**: Crowdsourcing allows for continuous refinement of translations, as contributors can suggest corrections and updates over time, improving content accuracy and consistency.
+
+## Adding Crowdsourcing Step to Your Workflow
 
-## General
+You can add Crowdsourcing to your project workflow in the workflow editor or the Workflow template editor.
 
-To get started, open your project and select **Crowdsourcing** on the left sidebar. In the *General* section, you can change the public project name, logo, and description. You can also change the public project URL, which you'll share with the community.
+<Image src={crowdsourcingWorkflowEditor} alt="Crowdsourcing in the Workflow Editor" />
```
[the remark this answers is not in the document]
Comment (rohalskyy) [reply src/content/docs/enterprise/organization-management/crowdsourcing.mdx:43]: We already have the LinkCard to the Workflow Overview after the intro paragraph. Wouldn't the "read more" to the Workflow Overview be redundant here?

File: src/content/docs/enterprise/organization-management/crowdsourcing.mdx [hunk 3]
```diff
@@ -7,62 +7,124 @@ slug: enterprise/crowdsourcing
...
+
+After adding the Crowdsourcing workflow step to your project, you can configure the following parameters:
+
+* Name &ndash; Set a custom name for the Crowdsourcing step. The default name is **Crowdsourcing**.
+* Output &ndash; Select the condition of the string to move to the next workflow step. You can choose between **Translated string** and **Translated string with minimum rating of**. For the latter, you can set the required minimum rating for the string before it moves to the next workflow step.
+* Languages &ndash; Select the languages that need translation.
+
+## Crowdsourcing Project Settings
+
+To start configuring your project's Crowdsourcing settings, open your project and go to Workflow > Crowdsourcing. Here, you can manage key aspects, including general project details, Readme content for contributors, status images reflecting localization progress, and the project link to build your community.
```
[the remark this answers is not in the document]
Comment (rohalskyy) [reply src/content/docs/enterprise/organization-management/crowdsourcing.mdx:54]: We are. Most likely, the formatting was lost during one of the text reviews. Fixed that

File: src/content/docs/enterprise/organization-management/crowdsourcing.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrii-bodnar) [reply src/content/docs/enterprise/organization-management/crowdsourcing.mdx:43]: Okay, let's leave it as is

**Review by andrii-bodnar — APPROVED** [review]

**Pull request merged**